app/code/Magento/Inventory/Model/Resource/SourceCarrierLink/Collection.php

```diff
@@ -0,0 +1,36 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace Magento\Inventory\Model\Resource\SourceCarrierLink;
+
+use Magento\Inventory\Model\Resource\SourceCarrierLink as ResourceSourceCarrierLink;
+use Magento\InventoryApi\Api\Data\SourceCarrierLinkInterface;
+use Magento\Framework\Model\ResourceModel\Db\Collection\AbstractCollection;
+use Magento\Inventory\Model\SourceCarrierLink as SourceCarrierLinkModel;
+
+class Collection extends AbstractCollection
```

Comment:
why do we need SourceCarrierLink\Collection? 
I would prefer not to add this class now, while we don't have specific use-cases for it